libs/vscode-ui/components/src/lib/select/select.component.spec.ts

```diff
@@ -1,21 +1,54 @@
+import { Component, ViewChild } from '@angular/core';
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { FormBuilder, ReactiveFormsModule } from '@angular/forms';
 import { OptionType } from '@angular/cli/models/interface';
 import { Option, OptionComponent } from '@nx-console/schema';
 import { SelectComponent } from './select.component';
```

Comment:
TODO: I am going to have to update the other vscode-ui-component specs and get them running in another pass on this.